### DISCUSSION TITLE
axis settings

### DISCUSSION THREAD

**Discussion by PBJohnsen** [artisan-roaster-scope/artisan#1004] [Q&A]
A while ago I had to reinstall Artisan because of a corrupt file. While it is working fine, I didn't save my settings from the former installation. When you have the cursor in the graph field a readout of the axes shows above the graph. Right now it is time and temp, but the former install showed time and ROR. is there  a setting to get the ROR back instead of temp?

**Comment by bigl00z3** [comment]
You can switch what is being displayed by pressing the letter "D". For all the shortcuts, go to Help/Keyboard Shortcuts.

I also recommend to watch the videos by Dave [here](https://youtu.be/tJKRJrEPeEQ) and [here](https://youtu.be/8ivsccu9e_Y), so many useful features are explained making roasting with Artisan an easier experience.

Nico

**Comment by PBJohnsen** [comment]
Thank you

On Sun, Nov 6, 2022 at 2:34 PM Nico Bigler ***@***.***> wrote:

> You can switch what is being displayed by pressing the letter "D". For all
> the shortcuts, go to Help/Keyboard Shortcuts.
>
> I also recommend to watch the videos by Dave here
> <https://youtu.be/tJKRJrEPeEQ> and here <https://youtu.be/8ivsccu9e_Y>,
> so many useful features are explained making roasting with Artisan an
> easier experience.
>
> Nico
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1004#discussioncomment-4070616>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALNEDNXMMCLT4Q2FVDSJUNDWHAI33ANCNFSM6AAAAAARYSZG7A>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1004/comments/4070616@
> github.com>
>


-- 
Anyone can achieve their fullest potential, who we are might be
predetermined, but the path we follow is always of our own choosing
